qiskit_nature/circuit/library/ansatzes/utils/fermionic_excitation_generator.py

```diff
@@ -128,6 +124,21 @@ def generate_fermionic_excitations(
         The list of excitations encoded as tuples of tuples. Each tuple in the list is a pair. The
         first tuple contains the occupied spin orbital indices whereas the second one contains the
         indices of the unoccupied spin orbitals.
+
+    Examples:
+        Generate excitations with basic inputs.
+
+        >>> from qiskit_nature.circuit.library.ansatzes.utils.fermionic_excitation_generator import (\
+        generate_fermionic_excitations)
+        >>> generate_fermionic_excitations(num_excitations=1, num_spin_orbitals=6,\
+        num_particles=(1,1))
+        [((0,), (1,)), ((0,), (2,)), ((3,), (4,)), ((3,), (5,))]
```

Comment:
Thanks for updating this. I took a final look at the result - you can download the documentation artifact, i.e. the zip of the built content from CI, from the build results page if you did not know. It looks a bit off depending on page width

Here you see the leading space of the first line above (i.e. the import)
![image](https://user-images.githubusercontent.com/40241007/156836847-7b663623-1411-4eb8-a105-c9e99da1d993.png)

And in this one its ahead of the last parameter of the 2nd line in the function call.
![image](https://user-images.githubusercontent.com/40241007/156836975-43be3cad-b818-4e7d-9729-818cd4953571.png)

I am thinking perhaps it might be better to have the content on a single line for these 2 above that are continued and disable pylint line length check. What is happening in the output is that leading space in the continuation line is kept and can make the output look odd depending on page width and how things wrap.

I don't have any better idea other than disabling pylint check and letting the line be a little longer in the docs here. I think removing the leading whitespace as things are may well cause other issues with sphinx. Maybe you have seen that already.

Reply:
Ok I kept it all on one line and disabled the pylint check, screenshots of how it looks on different screen sizes. As I said earlier I don't think there is a perfect formatting solution using doctest, but I think this should be enough that users will understand at least

<img width="845" alt="Screenshot 2022-03-04 at 5 20 05 PM" src="https://user-images.githubusercontent.com/23662430/156851007-259edd83-4324-45ab-91e5-cdc4e55094be.png">
<img width="467" alt="Screenshot 2022-03-04 at 5 21 22 PM" src="https://user-images.githubusercontent.com/23662430/156851009-34766098-aa32-4c28-b2db-e501859cbc44.png">